docs/working-with-projects/working-with-images/preheat-images.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Preheat Images
+weight: 30
+---
+Preheat action is policy-based driven way, and the scope of preheat policy is limited in the project level. Preheat 
+policy created under the specified project can only preheat images manged under that project.
```

Comment:
This means that the preheat policy created under the specified project apply only to images manged under that project.